packages/byods/src/byods/index.ts

```diff
@@ -10,6 +10,10 @@ import {
 } from '../constants';
 import {SDKConfig, JWSTokenVerificationResult} from '../types';
 import TokenManager from '../token-manager';
+import {BYODSConfig} from '../token-manager/type';
```

Comment:
In BYODS class we are using types from token-mnager, We can either use the types that is created in byods class or we can have a common types file.